symengine/symengine_config_cling.h.in

```diff
@@ -0,0 +1,7 @@
+#ifndef SYMENGINE_CONFIG_CLING_HPP
+#define SYMENGINE_CONFIG_CLING_HPP
+
+#pragma cling add_library_path(@SYMENGINE_CLING_LIBRARY_DIR@)
+#pragma cling load("libsymengine")
```

Comment:
Is cling available on unixes only? If it's available on windows, this should be symengine

Reply:
We could use cmake's library prefix which is empty on windows, and basically do `{PREFIX}symengine`.

The stable version of cling does not function on windows, but master fixes those issues. Next version should have full windows support.

Reply:
Now making us of `CMAKE_SHARED_LIBRARY_PREFIX` to replace `lib`.